src/functionalTest/resources/features/F-001/S-010.td.json

```diff
@@ -34,6 +34,40 @@
             "substantive": "N"
           }
         },
+        {
```

Comment:
can we add sort criteria to the start of the array: 
```json 
{
        "__elementId__": "roleName"
},
```

Reply:
addressed in c94c50b